pkg/storage/sqlite/schema/schema.sql

```diff
@@ -93,14 +93,31 @@ CREATE TABLE IF NOT EXISTS "movie" (
     "last_search_time" DATETIME
 );
 
+CREATE TABLE IF NOT EXISTS "movie_transition" (
```

Comment:
we could probably get away with making the transitions one table, but wasn't sure if that was necessary or not

Reply:
I think series transitions might be more complicated depending on how we track it. Since there are episodes in seasons